lib/amo-client.js

```diff
@@ -151,6 +148,76 @@ Client.prototype.waitForSignedAddon = function(statusUrl, opt) {
 };
 
 
+Client.prototype.downloadSignedFiles = function(signedFiles, options) {
+  options = _.assign({
+    createWriteStream: fs.createWriteStream,
+    request: this._request,
+    stdout: process.stdout,
+  }, options);
+  var self = this;
+  var allDownloads = [];
+  // Set some defaults so it looks like 0% before starting the download.
+  var dataExpected = 100;
+  var dataReceived = 0.1;
```

Comment:
I think I'd rather see the `if` and `Waiting for download to start...` before it starts. I think you could set `dataExpected = 0.00001` and `dataReceived = 0` if you don't want to do that though.